Resources/config/services.xml

```diff
@@ -223,6 +225,16 @@
             <tag name="sulu.context" context="admin"/>
         </service>
 
+        <!-- Form Website Controller -->
+        <service id="sulu_form.form_website_controller" class="Sulu\Bundle\FormBundle\Controller\FormWebsiteController" public="true">
```

Comment:
Can't tag this controller with `sulu.context` `website`, because otherwise i wouldn't be able to alias it. @alexander-schranz wdyt?

Reply:
Do we need to register it as a service as I think the `Sulu\Bundle\WebsiteBundle\Controller\DefaultController` is also no service or? What happens if we remove the service definition?

We should split the controller `Website/FormStaticController` and `Website/FormTokenController`.

Reply:
Yes it is.
```xml
<service id="sulu_website.default_controller" class="Sulu\Bundle\WebsiteBundle\Controller\DefaultController">
    <tag name="container.service_subscriber" />
    <tag name="controller.service_arguments" />
    <tag name="sulu.context" context="website" />
    <call method="setContainer">
        <argument type="service" id="Psr\Container\ContainerInterface" />
    </call>
</service>
<service id="Sulu\Bundle\WebsiteBundle\Controller\DefaultController" alias="sulu_website.default_controller"/>
```
This is the definition of sulu's `DefaultController`. But I have absolutely no idea why it's working here, that the controller is tagged with `sulu.context` `website`.

Reply:
If I remove the alias, I get the same error as in #233

Reply:
Have a look at my latest commit, this seems to solve it. Maybe it's working for the `DefaultController`, because neither the service nor the alias is public. But if I remove the public attribute from the `FormWebsiteController` and it's alias, I also get the same error as in #233.